packages/ckeditor5-clipboard/src/dragdropexperimental.ts

```diff
@@ -583,40 +584,32 @@ export default class DragDropExperimental extends Plugin {
 
 				widgetToolbarRepository.forceDisabled( 'dragDrop' );
 			}
+
+			return;
 		}
 
 		// If this was not a widget we should check if we need to drag some text content.
-		else if ( !selection.isCollapsed || ( selection.getFirstPosition()!.parent as Element ).isEmpty ) {
+		if ( !selection.isCollapsed || ( selection.getFirstPosition()!.parent as Element ).isEmpty ) {
 			const blocks = Array.from( selection.getSelectedBlocks() );
```

Comment:
@niegowski I'm wondering if there's a case where `getSelectedBlocks()` can return an empty array, given that we've already checked for collapsed selection.